dds/DCPS/DiscoveryBase.h

```diff
@@ -1380,12 +1384,13 @@ namespace OpenDDS {
           already_matched = lpi->second.matched_endpoints_.count(reader);
           writer_type_info = &lpi->second.type_info_;
           topic_name = topic_names_[lpi->second.topic_id_];
-
+          writer_participant_discovered_at = lpi->second.participant_discovered_at_;
```

Comment:
"participant discovered at" seems like it would not be relevant for a local publication

Reply:
The logging creates a timeline starting at participant discovery.  For local entities that may associate with each other, the time since discovery that is logged will accurately record the latency from participant creation to the association of local endpoints.